appservice.go

```diff
@@ -18,5 +18,5 @@ package gomatrixserverlib
 // ApplicationServiceTransaction is the transaction that is sent off to an
 // application service.
 type ApplicationServiceTransaction struct {
-	Events []Event `json:"events"`
+	Events []*Event `json:"events"`
```

Comment:
We don't need a slice of pointers here... it's already passed by reference due to the slice.